pallet/account-migration/src/lib.rs

```diff
@@ -129,7 +129,9 @@ pub mod pallet {
 	pub enum Event {
 		/// An account has been migrated.
 		Migrated { from: AccountId32, to: AccountId20 },
-		/// An multisig account has been migrated.
+		/// A new multisig account params was noted/recorded on-chain.
+		NewMultisigParamsNoted { from: AccountId32, to: MultisigParams },
```

Comment:
Suggest renaming `to: MultisigParams` to `param: MultisigParams`. The struct contain more than just `to` information.

Reply:
`param` looks confusing to me. The param is all about the `to`, the new multisig.